src/gui/optionsdlg.ui

```diff
@@ -2420,6 +2420,11 @@
                    <string>Remove them</string>
                   </property>
                  </item>
+                 <item>
+                  <property name="text">
+                   <string>Super seed them</string>
```

Comment:
Is it correct wording?

Reply:
I'm not sure. At first i had " Enable super seeding for them" but it seems excessively long.

Reply:
@LordNyriox any ideas?

Reply:
@thalieht:  I do not even understand what `them` refers to.  

A pronoun without an antecedent is confusing as heck.  

That being said, *if* `them` refers to torrents I would write `Use super seeding mode for them`—since `super seeding` in this case is a mode, rather than an action.

Reply:
@thalieht:  `Enable super seeding for them` works as well—since using the word "enable" implies a binary state (AKA, a "mode").

Reply:
@thalieht:  `Super seed them`, on the other hand, does NOT work—since it implies that the actual action being performed is "super seeding" itself, rather than "enabling super seeding mode".  

English is fraught with pitfalls like that.

Reply:
Sorry, indeed 'them' refers to torrents. Thanks! Guess i'll go with "Enable super seeding for them" unless there are further objections.